src/main/scala/com/databricks/spark/redshift/RedshiftJDBCWrapper.scala

```diff
@@ -222,7 +225,13 @@ private[redshift] class JDBCWrapper {
     }.getOrElse {
       throw new IllegalArgumentException(s"Did not find registered driver with class $driverClass")
     }
-    driver.connect(url, new Properties())
+    val properties = new Properties()
+    credentials.foreach { case(user, password) => {
+        properties.setProperty("user", user)
```

Comment:
According to codecov, it looks like this was never actually hit during tests. I'm going to go ahead and merge this now and will add an end-to-end test myself in a followup patch.